oracle/config/app.go

```diff
@@ -65,6 +85,12 @@ type AppConfig struct {
 
 	// MetricsEnabled is a flag that determines whether oracle metrics are enabled.
 	MetricsEnabled bool `mapstructure:"metrics_enabled" toml:"metrics_enabled"`
+
+	// MaxAge is the maximum age of the latest price response before it is considered stale.
+	MaxAge time.Duration `mapstructure:"max_age" toml:"max_age"`
```

Comment:
`oracle.max_age` feels pretty vague in terms of describing what this does. How about `oracle.price_ttl` instead or something along those lines?